addons/survey/views/survey_survey_views.xml

```diff
@@ -76,11 +76,15 @@
                                     <field name="suggested_answer_ids" invisible="1"/>
                                     <field name="matrix_row_ids" invisible="1"/>
                                     <field name="survey_id" invisible="1"/>
-                                    <field name="triggering_question_id" invisible="1"/>
                                     <field name="random_questions_count"
                                         attrs="{'column_invisible': [('parent.questions_selection', '=', 'all')], 'invisible': [('is_page', '=', False)]}"/>
-                                    <button disabled="disabled" icon="fa-code-fork" attrs="{'invisible': [('triggering_question_id', '=', False)]}"
-                                        title="This question depends on another question's answer." class="icon_rotates"/>
+                                    <field name="triggering_question_id" invisible="1"/>
+                                    <field name="triggering_answer_id" invisible="1"/>
+                                    <widget name="survey_question_trigger" options="{'isSurveyForm': True}"/>
+                                    <!-- Ensure these fields are synced with db too when opening a question -->
+                                    <field name="is_conditional" invisible="1"/>
+                                    <field name="is_placed_before_trigger" invisible="1"/>
+                                    <field name="allowed_triggering_question_ids" invisible="1"/>
```

Comment:
Would put them above and remove the comment after all, it's just packing them with all the other invisible and they all have the same purpose, so we probably don't need a different comment.